string type
string Name
float32 ini_pose_x
float32 ini_pose_y
float32 ini_pose_z
